synapseclient/core/version_check.py

```diff
@@ -163,16 +274,37 @@ def _version_tuple(version, levels=2):
     return tuple(v)
 
 
-def _get_version_info(version_url=_VERSION_URL):
+def _get_version_info(version_url: Optional[str] = _VERSION_URL) -> dict:
```

Comment:
Is there a more specific Dictionary we can update the return type to? Like a Dict[str, str]?

Reply:
This function was removed